### PR TITLE
Preparation for Key Vault Keys beta release for August 2021

### PR DESCRIPTION
**Pull request by vcolin7**

### CODE REVIEW

**Comment by vcolin7** [comment]
@srnagar @alzimmermsft I think an update to RevApi might be causing [this error](https://dev.azure.com/azure-sdk/public/_build/results?buildId=1044455&view=logs&j=b70e5e73-bbb6-5567-0939-8415943fadb9&t=ba3cf43a-b2bf-50f9-f95f-93049ca44ad1&l=804), as we made this change a while ago and it had not complained until now. Also, the classes it's talking about had always been package private and were moved to a package where they make more sense to be, I don't think this is a breaking change. Should we just add an exception in this case?

File: sdk/keyvault/azure-security-keyvault-keys/CHANGELOG.md [hunk 1]
```diff
@@ -1,10 +1,19 @@
...
 ### Features Added
-- Added `Exportable` and `ReleasePolicy` to `CreateKeyOptions`, `ImportKeyOptions`, and `KeyProperties` to support Secure Key Release for Key Vault and Managed HSM.
-- Added a `release()` operation to `KeyClient` and `KeyAsyncClient` to release a key for Key Vault and Managed HSM.
+- To support Secure Key Release for Key Vault and Managed HSM, added `Exportable` and `ReleasePolicy` to the following classes:
+  - `CreateKeyOptions` and its children classes: `CreateEcKeyOptions`, `CreateOctKeyOptions` and `CreateRsaKeyOptions`.
+  - `ImportKeyOptions`
+  - `KeyProperties`
+- Added `releaseKey()` and `releaseKeyWithResponse()` operations to `KeyClient` and `KeyAsyncClient` to securely release a key for Key Vault and Managed HSM.
+
+### Other Changes
```
Comment (srnagar) [inline sdk/keyvault/azure-security-keyvault-keys/CHANGELOG.md:12]: Remove empty section

**Review by srnagar — APPROVED** [review]

**Comment by srnagar** [comment]
@vcolin7 If a package-private class was moved from one package to another, then it's not a breaking change. If revapi flags it, then you can add a suppression. 
Please link the PR that moved `Base64UrlJsonDeserializer` and `Base64UrlJsonSerializer` to a different package.

File: eng/code-quality-reports/src/main/resources/revapi/revapi.json [hunk 2]
```diff
@@ -340,6 +340,42 @@
...
+        "attribute": "using",
+        "oldValue": "com.azure.security.keyvault.keys.models.Base64UrlJsonDeserializer.class",
+        "newValue": "com.azure.security.keyvault.keys.implementation.Base64UrlJsonDeserializer.class",
+        "package": "com.azure.security.keyvault.keys.models",
+        "classQualifiedName": "com.azure.security.keyvault.keys.models.JsonWebKey",
+        "classSimpleName": "JsonWebKey",
+        "methodName": "getDq",
+        "oldArchive": "com.azure:azure-security-keyvault-keys:jar:4.3.1",
+        "newArchive": "com.azure:azure-security-keyvault-keys:jar:4.4.0-beta.2",
+        "elementKind": "method"
```
Comment (alzimmermsft) [inline eng/code-quality-reports/src/main/resources/revapi/revapi.json:360]: nit: only need to incluide `code`, `new` and `justification`

File: eng/code-quality-reports/src/main/resources/revapi/revapi.json [hunk 3]
```diff
@@ -340,6 +340,42 @@
         "code": "java.method.added",
         "new": "method com.azure.core.http.policy.HttpLogOptions com.azure.core.http.policy.HttpLogOptions::setResponseLogger(com.azure.core.http.policy.HttpResponseLogger)",
         "justification": "New method included in azure-core."
+      },
+      {
+        "code": "java.annotation.attributeValueChanged",
+        "old": "method byte[] com.azure.security.keyvault.keys.models.JsonWebKey::getDq()",
```
Comment (alzimmermsft) [inline eng/code-quality-reports/src/main/resources/revapi/revapi.json:346]: There should be another pattern already suppression a bunch of JsonWebKey API additions, you could add this to that.

File: sdk/keyvault/azure-security-keyvault-keys/CHANGELOG.md [hunk 4]
```diff
@@ -1,10 +1,19 @@
 # Release History
 
-## 4.4.0-beta.2 (Unreleased)
+## 4.4.0-beta.2 (2021-08-10)
```
Comment (alzimmermsft) [inline sdk/keyvault/azure-security-keyvault-keys/CHANGELOG.md:3]: nit: Wrong date

File: eng/code-quality-reports/src/main/resources/revapi/revapi.json [hunk 5]
```diff
@@ -340,6 +340,42 @@
...
+        "new": "method byte[] com.azure.security.keyvault.keys.models.JsonWebKey::getDq()",
+        "justification": "Annotation moved to different package was package-private.",
```
Comment (srnagar) [inline eng/code-quality-reports/src/main/resources/revapi/revapi.json:348]: Re-word this to clarify that this isn't a breaking change - "Package-private Base64UrlJsonDeserializer class moved from models package to implementation package Base64UrlJsonDeserializer"

File: eng/code-quality-reports/src/main/resources/revapi/revapi.json [hunk 6]
```diff
@@ -340,6 +340,42 @@
...
+        "methodName": "getDq",
+        "oldArchive": "com.azure:azure-security-keyvault-keys:jar:4.3.1",
+        "newArchive": "com.azure:azure-security-keyvault-keys:jar:4.4.0-beta.2",
+        "elementKind": "method"
+      },
+      {
+        "code": "java.annotation.attributeValueChanged",
+        "old": "method byte[] com.azure.security.keyvault.keys.models.JsonWebKey::getDq()",
+        "new": "method byte[] com.azure.security.keyvault.keys.models.JsonWebKey::getDq()",
+        "justification": "Annotation moved to different package was package-private.",
```
Comment (srnagar) [inline eng/code-quality-reports/src/main/resources/revapi/revapi.json:366]: Same here - update the description

**Review by g2vinay — APPROVED** [review]

**Pull request merged**